src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/RazorVersionedTextDocumentIdentifier.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
+
+internal record class RazorVersionedTextDocumentIdentifier(TextDocumentIdentifier TextDocumentIdentifier, int Version);
```

Comment:
It would be remiss of me not to mention that I'm not a fan of this name, but it is bike shedding in the extreme, since essentially nobody will see it. I think `TextDocumentIdentifierAndVersion` would be better 😛

Reply:
I couldn't think of a good name so I just made a verbose one. `TextDocumentIdentifierAndVersion` works for me!